internal/pkg/manifest/workload.go

```diff
@@ -69,10 +69,10 @@ var (
 
 	defaultPlatform = platformString(OSLinux, ArchAMD64)
 
-	windowsTaskCPU    = 1024
-	windowsTaskMemory = 2048
+	WindowsTaskCPU    = 1024
+	WindowsTaskMemory = 2048
```

Comment:
nit:
```suggestion
	MinWindowsTaskCPU    = 1024
	MinWindowsTaskMemory = 2048
```

Reply:
Doc comments please for all exported names 🥺 
Also can we move the exported names above the private ones?